state/protocol/invalid/snapshot.go

```diff
@@ -89,3 +93,23 @@ func (u *Snapshot) ProtocolState() (protocol.KVStoreReader, error) {
 func (u *Snapshot) VersionBeacon() (*flow.SealedVersionBeacon, error) {
 	return nil, u.err
 }
+
+type EpochQuery struct {
```

Comment:
Since we are adding a new type a short go doc would be appreciated to state the purpose.